seqr/utils/file_utils.py

```diff
@@ -81,17 +81,19 @@ def mv_file_to_gs(local_path, gs_path, user=None):
     _run_gsutil_with_wait(command, gs_path, user)
 
 
-def get_gs_file_list(gs_path, user=None):
+def get_gs_file_list(gs_path, user=None, check_subfolders=True):
     gs_path = gs_path.rstrip('/')
-    command = 'ls'
+    command = '-q ls'
 
     # If a bucket is empty gsutil throws an error when running ls with ** instead of returning an empty list
     subfolders = _run_gsutil_with_wait(command, gs_path, user, get_stdout=True)
     if not subfolders:
         return []
 
-    all_lines = _run_gsutil_with_wait(command, f'{gs_path}/**', user, get_stdout=True)
-    return [line for line in all_lines if line.startswith(gs_path)]
```

Comment:
the `if line.startswith(gs_path)` is necessary to omit any warnings that are logged to the console

Reply:
Hm... you are right.

Reply:
But the filter doesn't work to get the sharded VCF files because no files have `*` in their names. We could filter by either starting with `gs://` or ending with VCF extensions. Maybe I should change the function name from `get_gs_file_list` to `get_vcf_file_list` and add filtering by the VCF extensions.

Reply:
ah I see. Lets just change `line.startswith(gs_path)` to ` if is_google_bucket_file_path(line)`